x/oracle/keeper/migrations.go

```diff
@@ -21,3 +22,9 @@ func (m Migrator) Migrate1to2(ctx sdk.Context) error {
 	v2.UpdateParams(ctx, m.keeper.paramSpace)
 	return v2.MigrateTaskStore(ctx, m.keeper.storeKey, m.keeper.cdc)
 }
+
+// Migrate2to3 migrates from version 2 to 3.
+func (m Migrator) Migrate2to3(ctx sdk.Context) error {
```

Comment:
I saw the value under `Withdraw` key also has string formatted address, do we need to migrate them as well?

Reply:
At x/oracle/types/withdraw.go:L12